lib/presentation/widget/shared_space/shared_space_viewmodel.dart

```diff
@@ -89,18 +97,52 @@ class SharedSpaceViewModel extends BaseViewModel {
                   store.dispatch((SharedSpaceSetSearchResultAction(_sharedSpaceNodes)));
                   _searchQuery = SearchQuery('');
                 } else if (success is CreateWorkGroupViewState) {
-                  getAllSharedSpaces();
+                  getAllSharedSpaces(needToGetOldSorter: false);
```

Comment:
Why not returning always with last sorter ?

Reply:
Because the list will be kept with the old sorter after adding or deleting items. We only need to fetch the saved sorter when the first time opens the screen.